examples/algorithms/plot_trust_rank.py

```diff
@@ -0,0 +1,62 @@
+"""
+===================
+TrustRank Algorithm
+===================
```

Comment:
Since this is in the `Algorithms` section, the `Algorithm` in the title name is a bit redundant

```suggestion
=========
TrustRank
=========
```